lib/fathead/mongoosejs/output.txt

```diff
@@ -0,0 +1,590 @@
+Document-get	A			Document						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Document#get(path, [type])</code></pre><p>Returns the value of a path.</p><span class="prog__sub">Parameters:</span><ul><li><p><code>path</code> <code>String</code> </p></li><li><p><code>[type]</code> <code>Schema, String, Number, Buffer, *</code> optionally specify a type for on-the-fly attributes</p></li></ul><span class="prog__sub">Example</span><pre><code>// path\ndoc.get('age') // 47\n\n// dynamic casting to a string\ndoc.get('age', String) // "47"</code></pre></section>	http://mongoosejs.com/docs/api.html#document_Document-get
+Document-init	A			Document						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Document#init(doc, fn)</code></pre><p>Initializes the document without setters or marking anything modified.</p><span class="prog__sub">Parameters:</span><ul><li><p><code>doc</code> <code>Object</code> document returned by mongo</p></li><li><p><code>fn</code> <code>Function</code> callback</p></li></ul><p>Called internally after a document is returned from mongodb.</p></section>	http://mongoosejs.com/docs/api.html#document_Document-init
+Document-inspect	A			Document						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Document#inspect()</code></pre><p>Helper for console.log</p></section>	http://mongoosejs.com/docs/api.html#document_Document-inspect
+Document-invalidate	A			Document						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Document#invalidate(path, errorMsg, value, [kind])</code></pre><p>Marks a path as invalid, causing validation to fail.</p><span class="prog__sub">Parameters:</span><ul><li><p><code>path</code> <code>String</code> the field to invalidate</p></li><li><p><code>errorMsg</code> <code>String, Error</code> the error which states the reason path was invalid</p></li><li><p><code>value</code> <code>Object, String, Number, T</code> optional invalid value</p></li><li><p><code>[kind]</code> <code>String</code> optional kind property for the error</p></li></ul><span class="prog__sub">Returns:</span><ul><li><p><code></code> <code>ValidationError</code> the current ValidationError, with all currently invalidated paths</p></li></ul><p>The erro
```

Comment:
@moollaza What should the instant answer be when a user types  `mongoose skip`?

---

Articles are `Query-skip`, `Aggregate-skip`. 

**Example Queries** 1) `mongoose skip` 2) `mongoose query skip` 3) `mongoose aggregate skip` 

**No Redirects**
`Query-skip A`, `Aggregate-skip A`

**Redirect**
`Query-skip A`, `Aggregate-skip A`, `skip R Query-skip` OR `skip R Aggregate-skip`

**Disambiguation**
`Query-skip A`, `Aggregate-skip A`, `skip D *[[Query-skip]]*/n[[Aggregate-skip]]`.

**Result**
- **No Redirects** - 1) Nothing 2) `Query-skip A`  3) `Aggregate-skip A`
- **Redirect** - 1) `Aggregate-skip A` OR `Query-skip A` 2) `Query-skip A` 3) `Aggregate-skip A`
 - **Disambiguation** - 1) `Query-skip A`, `Aggregate-skip A` 2) `Query-skip A` 3) `Aggregate-skip A`

Disambiguation wins.

This is it, right? or am I mistaken?